setup.py

```diff
@@ -30,7 +30,7 @@
         'aioredis',
         'aiozk',
         'async_timeout',
-        'dash',
+        'dash>=1.18.*',
```

Comment:
Not sure that `*` makes logical sense with `>=`, but presumably pip is happy enough or this wouldn't have built). I would have said `dash>=1.18`.

Reply:
noted